requirements-base.txt

```diff
@@ -0,0 +1,15 @@
+great_expectations==0.11.7
```

Comment:
im kinda guessing on what is needed  and what is "extra"

Reply:
No worries. I'll take care of it.